Makefile

```diff
@@ -193,11 +193,13 @@ local-docker-rebuild:
 	docker-compose up -d quay-db --build
 	docker exec -it quay-db bash -c 'while ! pg_isready; do echo "waiting for postgres"; sleep 2; done'
 	docker-compose up -d quay --build
+	docker-compose restart quay
```

Comment:
`--force-recreate` [might be worth checking out to save line. ](https://docs.docker.com/engine/reference/commandline/compose_up/)

Reply:
I originally had `--force-recreate` in there, but there's something odd with it and the caching. It would always trigger a rebuild for these layers:

```
 => CACHED [build-static 2/5] RUN set -ex ; mkdir -p static/{webfonts,fonts,ldn} ; python3 -m external_libraries ;                                                                               0.0s
 => [build-static 3/5] COPY --chown=0:0 static/  ./static/                                                                                                                                       0.2s
 => [build-static 4/5] COPY --chown=0:0 *.json *.js  ./                                                                                                                                          0.0s
 => [build-static 5/5] RUN set -ex ; npm install --quiet --no-progress --ignore-engines ; npm run --quiet build ;                                                                               88.9s
 => CACHED [local-dev-env 1/6] RUN alternatives --set python /usr/bin/python3                                                                                                                    0.0s
 => CACHED [local-dev-env 2/6] WORKDIR /quay-registry
 ```

Reply:
Interesting, I wonder why. No problem.